R/join_key.R

```diff
@@ -12,9 +11,9 @@
 #' where `names(keys)` maps columns in `dataset_1` corresponding to columns of
 #' `dataset_2` given by the elements of `keys`.
 #'
-#' If unnamed, the same column names are used for both datasets.
+#' * If unnamed, the same column names are used for both datasets.
 #'
-#' If any element of the `keys` vector is empty with a non-empty name, then the name is
+#' * If any element of the `keys` vector is empty with a non-empty name, then the name is
```

Comment:
Nice to use unordered list for this one :+1: 

Maybe remove empty line between them.

Consistency in markdown: Can you use `-` as below and as the rest of the documentation (either that or replace all with `-`)

````suggestion
#' - If unnamed, the same column names are used for both datasets.
#' - If any element of the `keys` vector is empty with a non-empty name, then the name is
````

Reply:
I will replace this with a hyphen (-) as it is used in many places.